src/lib.rs

```diff
@@ -17,6 +17,10 @@ extern crate derivative;
 #[macro_use]
 extern crate pest_derive;
 
+#[cfg(feature = "api-client")]
+#[macro_use]
+extern crate prettytable;
```

Comment:
So you generally don't need to do this. You can do things like `#[prettytable::Thingy]` decorators and use `use prettytable::DerivableTrait` to import a custom derive. You may still need this though... Did you try without?

Reply:
This is for prettytable's macros: `row!`, `table!`, etc.

I did this Rust 2015-stylee because regular imports were throwing 'private mod' errors during dev (but not build), due to clashing between macro + mod names.

<img width="594" alt="Screenshot 2020-10-15 at 20 40 25" src="https://user-images.githubusercontent.com/719722/96178490-622a1b80-0f27-11eb-9e05-ad5a7795841f.png">

`use prettytable:*` would work here too, at the cost of some import pollution.

Do we have a preference?